azure-functions/acbs-function/src/functions/acbs-amend-facility-loan-record.js

```diff
@@ -38,11 +38,22 @@ df.app.orchestration('acbs-amend-facility-loan-record', function* amendFacilityL
 
     const { facilityId, facility, amendments, fmr } = payload;
     const { amendment } = amendments;
+
     let facilityLoanRecordAmendments;
 
     // 1.1. Facility Loan Record (FLR) amendment mapping
     const flrMApped = mappings.facility.facilityLoanAmend(amendments, facility, fmr);
 
+    /**
+     * When facility amendment is applied on facility type `Loan`,
+     * with only `amount` attribute execution should be terminated.
+     */
+    if (!flrMApped?.length) {
+      facilityLoanRecordAmendments = 'Facility loan amount only amendment, aborting FLR amendment';
```

Comment:
I'm not understanding the need/issue here, "with only amount attribute" - above comment will probably answer this and might suggest different wording here

Reply:
The above comment states the behaviour but I can elaborate here.
We only accepts two attributes at the moment on a facility amendment `cover end date` and `amount`, business requirements states that we should not process a loan (facility type) with `amount` only amendment.

Due to above requirement I have added a validation as without this validation mapping will be empty and APIM validation error will be thrown.

Reply:
Is this a short term fix? Are we allowing a user in TFM to create an amendment with just amount but then it fails when it gets to ACBS? Might be misunderstanding exactly how this is working so apologies if that's the case